plugins/tours/tests/phpunit/integration/template/renderPostmetaBeforeContent.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ *  Tests for render_postmeta_before_content()
+ *
+ * @since      1.0.0
+ * @author     Robert A. Gadon
+ * @package    spiralWebDb\CornerstoneTours\Template\Tests\Integration
+ * @link       http://spiralwebdb.com
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\CornerstoneTours\Template;
+
+use spiralWebDb\Cornerstone\Tests\Integration\Test_Case;
+use function spiralWebDb\CornerstoneTours\render_the_tour_regions;
+use function spiralWebDb\CornerstoneTours\render_tour_comments;
+
+/**
+ * @covers ::\spiralWebDb\CornerstoneTours\Template\render_postmeta_before_content
+ *
+ * @group tours
+ * @group template
+ */
+class Tests_RenderPostmetaBeforeContent extends Test_Case {
+
+	/**
+	 * @dataProvider addTestData
+	 */
+	public function test_should_render_postmeta_before_entry_content( $post_data, $post_meta, $expected_html ) {
+		$post    = $this->factory->post->create_and_get( $post_data );
+		$tour_id = $post->ID;
+
+		// Add post_meta so that it can be called during test assertion.
+		add_post_meta( $tour_id, 'tour_regions', $post_meta['tour_regions'] );
+		add_post_meta( $tour_id, 'tour_comments', $post_meta['tour_comments'] );
```

Comment:
You can simplify this code like this:

```php
foreach( $post_meta as $key => $meta ) {
   add_post_meta( $tour_id, $key, $meta );
}
```

This approach also allows you to add or decrease the meta needed for each test scenario in the test data. Makes your test more flexible. 

Remember, we want our tests to be flexible so that we can adjust the test data without having to change the test code.